pkg/kubelet/metrics/metrics.go

```diff
@@ -483,6 +483,16 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// ImagePullingCount is a gauge that tracks the number of Images currently pulling
+	ImagePullingCount = metrics.NewGauge(
+		&metrics.GaugeOpts{
+			Subsystem:      KubeletSubsystem,
```

Comment:
What additional value will this metric add, compared to the existing runtime_operation metric filtered by `operation_type==image_pull` ? Is there any use case where we need to know the in-flight image pull counts?